package day0116;

//강제예외 발생 throw
class UserException extends Exception{
	
	public UserException(String msg) {
		super(msg); //Exception클래스 생성자를 통해 정식 메세지로 등록
	}
}
///////////////////////
public class ThrowException_03 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
	}

}
